### ISSUE TITLE
Updated site following transition to OpenHW Foundation

### ISSUE THREAD

**Issue by shanda-eclipse** [openhwgroup/openhwgroup.org#729]
**Starting web content:** https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?usp=sharing

**Design:** https://xd.adobe.com/view/18c1c086-758b-421b-87eb-e278a97b887d-4d92/
Maira will provide the assets from https://gitlab.eclipse.org/eclipsefdn/marketing/-/issues/2694

**Code for membership form:** code snippet can be found in this issue: https://github.com/openhwgroup/openhwgroup.org/issues/728

Note: there is a lot of additional content still under discussion in a [separate document ](https://docs.google.com/document/d/1S5B_qmFkoYwrCe2jdGcwZBisANymeST2GocXiZOBl-E/edit?usp=sharing) but we would like to get this site started while we work out the details.

**Comment by shanda-eclipse** [comment]
Add the design files on Maira's behalf while she works on getting access to GitHub. 
[OpenHW-Web-Images.zip](https://github.com/user-attachments/files/18010115/OpenHW-Web-Images.zip)

**Comment by shanda-eclipse** [comment]
@oliviergoulet5 can we have the first iteration of this in early January?

I'm curious if anyone has requested/reserved a new domain name, something like openhwfoundation.org? @chrisguindon @eclipsewebmaster

**Comment by chrisguindon** [comment]
> I'm curious if anyone has requested/reserved a new domain name, something like openhwfoundation.org? @chrisguindon @eclipsewebmaster

As far as I know, I didn't hear anything related to that. @eclipsewebmaster?

**Comment by oliviergoulet5** [comment]
> @oliviergoulet5 can we have the first iteration of this in early January?

Considering that we also have other commitments in January, I find "early January" too tight of a window. I suggest January 30th with the homepage implementation as a first iteration. However, I'll do my best to finish it earlier if possible.

Would that work for you @shanda-eclipse?

**Comment by shanda-eclipse** [comment]
@chrisguindon @eclipsewebmaster thanks for confirming. I will likely open an infra next week to request a domain. 

@oliviergoulet5 based on what I know Mike and Paul would like this to be a priority in January, but I know we have lots going on, so just do your best.

**Comment by chrisguindon** [comment]
> @oliviergoulet5 based on what I know Mike and Paul would like this to be a priority in January, but I know we have lots going on, so just do your best.

Please don't hesitate to point them to me if they have any issues with what @oliviergoulet5 is proposing.

**Comment by shanda-eclipse** [comment]
When we launch this redesigned site, we would also like to update the domain name: https://gitlab.eclipse.org/eclipsefdn/infrazilla/-/issues/1764

**Comment by SharonCorbett** [comment]
During our internal launch call this morning, Executive raised the matter that this content needs to get published sooner rather than later as the outdated information is impacting clear communication and alignment for the membership process.
Paul White will review the preview and we can continue to adjust as we move forward.  

@chrisguindon Can we please see if we can adjust priority on this one.

Thanks,
Sharon

**Comment by chrisguindon** [comment]
> @chrisguindon Can we please see if we can adjust priority on this one.

Yep! @oliviergoulet5 has been prioritizing this task. I also spent some time this morning reviewing the remaining work to finalize an MVP as soon as possible.

**Comment by oliviergoulet5** [comment]
Hi everyone! Here is a preview of the redesign:

https://preview-730--openhwgroup-org.eclipsecontent.org/

I have a few questions and/or requests:

- What will we do with the [sidebar](https://www.openhwgroup.org/resources/openhwtv/)? Should we remove it or make its content more relevant to the OpenHW Foundation?
- Are we adding a CTA to the navbar like in the mockup? I would need the URL and text for that button
- I need the images for the Community News and Events on the homepage. @meKokabi (I'm unsure what Maira's GitHub username is so I can't ping)

Also, for the featured story, it still mentions OpenHW Group. As we launch the redesigned site, this must be updated via the newsroom. @shanda-eclipse

**Comment by shanda-eclipse** [comment]
> What will we do with the [sidebar](https://www.openhwgroup.org/resources/openhwtv/)? Should we remove it or make its content more relevant to the OpenHW Foundation?

The sidebar is no longer needed.

> Also, for the featured story, it still mentions OpenHW Group. As we launch the redesigned site, this must be updated via the newsroom.

+1 I will make that change

**Comment by shanda-eclipse** [comment]
Edits:

- [ ] In the jumbotron, the first button should be [Learn More](https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?tab=t.0#heading=h.4e199a31qn64)
- [ ] On https://preview-730--openhwgroup-org.eclipsecontent.org/membership/become-a-member/ can we move this content to the Jumbotron

![image](https://github.com/user-attachments/assets/56b08b24-1bfe-4fd2-a1ad-805e493a8111)


- [ ] On https://preview-730--openhwgroup-org.eclipsecontent.org/membership/become-a-member/ can we update the quotes to match [these](https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?tab=t.0#heading=h.1rk3raxm6iwa). I have removed references to OpenHW _Group_

- [ ] On https://preview-730--openhwgroup-org.eclipsecontent.org/get-involved/ we need an update the link for "View Calendar" This is the best option I found: https://calendar.google.com/calendar/u/0/r?cid=meetings@openhwgroup.org @Floadv298 do you have an alternate option?

- [ ] On https://preview-730--openhwgroup-org.eclipsecontent.org/get-involved/  the Subscribe button under the newsletter needs to be updated to https://eclipse.us6.list-manage.com/subscribe?u=eaf9e1f06f194eadc66788a85&id=e7538485cd

- [ ] On https://preview-730--openhwgroup-org.eclipsecontent.org/get-involved/ the text links need to be styled so they stand out ex
![image](https://github.com/user-attachments/assets/de905bcc-f965-4a90-b1e8-848c56e8375a)

- [ ] Can we add a jumbotron to https://preview-730--openhwgroup-org.eclipsecontent.org/resources/openhwtv/ so it matches all the other top level pages? And remove the sidebar as discussed in the previous comment

**Comment by SharonCorbett** [comment]
Re:  Also, for the featured story, it still mentions OpenHW Group. As we launch the redesigned site, this must be updated via the newsroom.

+1 I will make that change

Will that take care of the bottom of About Us which states "What's happening at OpenHW Group?
Find out the latest news for the OpenHW community. Read News"??

**Comment by oliviergoulet5** [comment]
@SharonCorbett Yes that's correct

**Comment by SharonCorbett** [comment]
I've captured a few other things in by way of comment/suggestions in https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?tab=t.0

**Comment by oliviergoulet5** [comment]
> On https://preview-730--openhwgroup-org.eclipsecontent.org/membership/become-a-member/ can we update the quotes to match [these](https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?tab=t.0#heading=h.1rk3raxm6iwa). I have removed references to OpenHW Group

@shanda-eclipse I can make this change. However, when you click "View More", we have many more testimonials that need to be updated.

**Comment by oliviergoulet5** [comment]
I have updated the preview with all the changes requested @shanda-eclipse @SharonCorbett 

**Meta Description**
Also, I made a quick change to the meta description from

> OpenHW Group is a not-for-profit, global organization driven by its members and individual contributors where hardware and software designers collaborate on …

to 

> OpenHW Foundation is an Eclipse Foundation Working Group driven by its members and individual contributors where hardware and software designers collaborate on open source cores, related IP, tools and software projects.

But I'm sure you'll want to reword it.

**Orphan Pages**

I'll list all the orphan pages that are no longer linked (but still indexed by search engines) at a later time. Then, we can discuss what URLs need redirects.

We can probably do this after launch if we're pressed for time.

**Comment by shanda-eclipse** [comment]
> > On https://preview-730--openhwgroup-org.eclipsecontent.org/membership/become-a-member/ can we update the quotes to match [these](https://docs.google.com/document/d/1CsrgA_wr4LADbndPRudlcbpVVEhQJUm-LnrQGeti7rk/edit?tab=t.0#heading=h.1rk3raxm6iwa). I have removed references to OpenHW Group
> 
> @shanda-eclipse I can make this change. However, when you click "View More", we have many more testimonials that need to be updated... unless we only want to display the three?

Let's just edit the 3 that are visible for now, as to not delay the launch. I will work on editing the rest later

**Comment by shanda-eclipse** [comment]
@croundy @Floadv298 would you like to [review staging](https://preview-730--openhwgroup-org.eclipsecontent.org/) before we push this live? I know there is some urgency around the launch, so we will continue to refine and expand the website once it's live.

**Comment by croundy** [comment]
A few things I noticed while reviewing on my phone:

1. Jumbotron CTA text boxes have squared corners, rather than rounded like the rest of the text boxes.
2. the News button doesn't work
3. The text boxes under the "Why Join" section don't line up.
4. Some graphics and text don't seem to scale proportionally

![Image](https://github.com/user-attachments/assets/ab3553b5-9880-4806-9435-08c55ecc3016)

![Image](https://github.com/user-attachments/assets/5c4f0ec8-4a56-44ad-a73e-a1b26af828dd)

![Image](https://github.com/user-attachments/assets/b4fe79fe-e7c3-4539-9e0e-266929563083)

**Comment by oliviergoulet5** [comment]
@shanda-eclipse We got the following feedback during code review:

> The only things I see are the megamenu dropdowns aren't styled beyond the default... We may want to see if we need something different than 2 grey tones there...

Are we going to keep the base colours for the menus?

**Comment by shanda-eclipse** [comment]
> [@croundy](https://github.com/croundy) These should now be fixed, except for the Read News. That bad URL is coming from the Newsroom. Can you take a look [@shanda-eclipse](https://github.com/shanda-eclipse)?

I'm getting an error saying newsroom is not available, so I can't double check. But the issue is likely that we are pointing to the new URL that isn't live yet, rather then the path for news on the old site.

**Comment by shanda-eclipse** [comment]
> [@shanda-eclipse](https://github.com/shanda-eclipse) We got the following feedback during code review:
> 
> > The only things I see are the megamenu dropdowns aren't styled beyond the default... We may want to see if we need something different than 2 grey tones there...
> 
> Are we going to keep the base colours for the menus?

@meKokabi can you make a recommendation?

**Comment by SharonCorbett** [comment]
We really really need to get this published.  Is there anything that is too horrible to not push that we can't adjust later?

**Comment by oliviergoulet5** [comment]
@SharonCorbett I can't speak for the content, but we should be good to go from a technical perspective.

However, considering this significant change to the website, I recommend going live on Monday unless necessary.

But if we do push live today with the redesign, we would definitely not want to change the domain until Monday since that involves more risk and would require multiple other teams to be involved.

**Comment by shanda-eclipse** [comment]
+1 for pushing live today or Monday

**Comment by oliviergoulet5** [comment]
@fredg02 @eclipsewebmaster FYI, we may need your assistance with this. I'll reach out when we try to change the primary domain from openhwgroup.org to openhwfoundation.org on Monday.

**Comment by SharonCorbett** [comment]
Yes, totally agree to not changing domains until Monday.  But if we can publish this today would be awesome! 
Have a great weekend!

**Comment by oliviergoulet5** [comment]
@SharonCorbett Have a great weekend to you as well!

To give an update:

The redesign is now live on the old www.openhwgroup.org domain.

I noticed Cloudflare is hiding the email from the "Get Involved" page as a way to protect the email from bots. I have never encountered this before, and it will unlikely be fixed today. 

Older pages like https://www.openhwgroup.org/working-groups/ can still be accessed by search engine results. I will look into implementing redirects and removing pages next week.

**Comment by PaulWhite-Eclipse** [comment]
Thanks to everyone for getting this new site up and running - it's appreciated. 

Can I suggest 2 immediate (not a fire, but would be very helpful) updates: 

1. On the Members page, can we add the following sentence (feel free to edit): 
"The following are the members of the OpenHW Foundation. This list does not include members of the original OpenHW Group that are in the process of migrating their membership to the new OpenHW Foundation. As members complete the migration process, their logos will be added here. Please use the Contact Us on the Become a Member page if you have questions or wish to complete the migration for your company." 

2. Resources tab. Any chance we can add "Events" to the top of the list, and link to events.eclipse.org. And on events.eclipse.org add an OpenHardware category. And note that Embedded World, FOSDEM, and most other events should be included. 

BTW, not meaning to pile on but the filters on that page are old and should be updated to list only the top 5-6, and with an "Other" listed to capture the rest. We do not need openMDM, Science, etc. listed.

**Comment by oliviergoulet5** [comment]
Hi @PaulWhite-Eclipse!

Here is a preview of the changes: https://preview-734--openhwgroup-org.eclipsecontent.org/

> On the Members page, can we add the following sentence

I placed the text in an alert. See [members page](https://preview-734--openhwgroup-org.eclipsecontent.org/membership/members/)

> Resources tab. Any chance we can add "Events" to the top of the list, and link to events.eclipse.org

I could link events.eclipse.org (now eclipse.org/events), but the user must manually apply the OpenHW filter once they arrive on the site. I can look into making the URLs for eclipse.org/events shareable with the filter state (e.g. `eclipse.org/events?categories=openhw`).

We also have the [events page](https://preview-734--openhwgroup-org.eclipsecontent.org/events/) on openhwgroup.org.

Until we get the stateful URLs, would you rather we link https://www.eclipse.org/events or the [event page](https://preview-734--openhwgroup-org.eclipsecontent.org/events/)?

**Comment by PaulWhite-Eclipse** [comment]
Membership page looks good - thanks for the quick turnaround. 

To your last question, either is fine with me - I'll defer to you or @shanda-eclipse if one is easier or better. But if we leave it as the old OpenHW Event page, can we please add Embedded World to it.

**Comment by shanda-eclipse** [comment]
I think leaving it as the OpenHW events page is fine. I will update the event in the newsroom so it is tagged for OpenHW.

**Comment by Floadv298** [comment]
Can we please add a link to our resources again.
It was under Projects previously, 
Links should be to our Github repository and readthedocs 
Currently it points to the Eclispe project and thats it. If people want to check the code or read the docuemntation, its hard to find

**Comment by oliviergoulet5** [comment]
I will close this issue since the original request has been fulfilled.

I created two issues to track some post-launch problems:

- https://github.com/openhwgroup/openhwgroup.org/issues/740
- https://github.com/openhwgroup/openhwgroup.org/issues/739
